application-admintools-api/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should merge this branch with the `main` one, to not display changes applied already